src/test/java/core/basesyntax/service/impl/PurchaseOperationTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperation;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PurchaseOperationTest {
```

Comment:
Test for negative values?

Reply:
that test in other test-cases (FruitRecordParserImplTest)